packages/components/src/__future__/Tag/Tag.tsx

```diff
@@ -26,33 +27,20 @@ export const Tag = ({
 }: TagProps): JSX.Element => (
   <span
     className={classNames(
-      "inline-flex items-center px-8 py-[5px] gap-4 rounded-[28px] font-family-paragraph text-paragraph-sm font-weight-paragraph leading-paragraph-sm whitespace-nowrap",
+      styles.container,
       {
-        ["bg-gray-200 text-purple-800"]: color === "gray",
-        ["bg-blue-100 text-blue-700"]: color === "blue",
-        ["bg-green-100 text-green-700"]: color === "green",
-        ["bg-yellow-100 text-yellow-700"]: color === "yellow",
-        ["bg-orange-100 text-orange-700"]: color === "orange",
-        ["bg-red-100 text-red-700"]: color === "red",
-        ["bg-purple-100 text-purple-700"]: color === "purple",
+        [styles.gray]: color === "gray",
+        [styles.blue]: color === "blue",
+        [styles.green]: color === "green",
+        [styles.yellow]: color === "yellow",
+        [styles.orange]: color === "orange",
+        [styles.red]: color === "red",
+        [styles.purple]: color === "purple",
       },
       classNameOverride
     )}
   >
-    {Icon && (
-      <span
-        className={classNames("inline-flex -my-[1px]", {
-          ["text-blue-500"]: color === "blue",
-          ["text-green-500"]: color === "green",
-          ["text-yellow-500"]: color === "yellow",
-          ["text-orange-500"]: color === "orange",
-          ["text-red-500"]: color === "red",
-          ["text-purple-500"]: color === "purple",
-        })}
-      >
-        {Icon}
-      </span>
-    )}
+    {Icon && <span className={styles.iconContainer}>{Icon}</span>}
```

Comment:
Suggestion: If the styles on the `iconContainer` are inconsequential, you can actually clone the passed in Icon component and use `classNameOverride` to add an `icon` class instead.

Only if you're keen to try though. The container method works fine too.